src/components/CharacterComponent.jsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import styled from "styled-components";
+
+import CharacterImg from "../shared/assets/fotoKim.jpeg";
+
+let scoreBar = "";
+let percentage = scoreBar;
```

Comment:
waarom declare je deze variabele?

Reply:
dit is nog oude code, heb ik in mijn commit hierna al verwijderd